static/app/components/group/suggestedOwners/suggestedOwners.spec.jsx

```diff
@@ -130,39 +128,6 @@ describe('SuggestedOwners', function () {
     );
   });
 
-  it('displays release committers', async function () {
```

Comment:
IDK what this release committers stuff but I guess it wasn't necessary?

Reply:
it never launched, i should've done it separate probably